src/stcal/jump/twopoint_difference.py

```diff
@@ -119,25 +117,41 @@ def find_crs(dataa, group_dq, read_noise, rejection_thresh,
     dnu_flag = dqflags["DO_NOT_USE"]
     jump_flag = dqflags["JUMP_DET"]
 
+    # get data, gdq
+    dat = dataa
+    num_flagged_grps = 0
+    # determine the number of groups with all pixels set to DO_NOT_USE
+    nints = nints
```

Comment:
??? what's the purpose of this?

Reply:
I thought I got paid per line of code. I removed it.